MODULE.bazel

```diff
@@ -5,11 +5,11 @@ module(
     repo_name = "rules_kotlin",
 )
 
-bazel_dep(name = "platforms", version = "0.0.6")
+bazel_dep(name = "platforms", version = "0.0.8")
```

Comment:
Can you also update the WORKSPACE versions of these so that they are kept somewhat in sync with each other?

Reply:
Sure. But I haven't found where `platforms` and `rules_cc` specified in WORKSPACE version.

Reply:
Updated PR

Reply:
Can you mirror these changes in `MODULE.release.bazel` so that the release archive also has these updates?

Reply:
Already did. Did I miss something?

Reply:
Nope, I just skimmed over it. Looks good to me.